specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/apimpolicies.json

```diff
@@ -58,13 +58,45 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "|     Field     |     Usage     |     Supported operators     |     Supported functions     |</br>|-------------|-------------|-------------|-------------|</br>| ComplianceState | filter | eq, ne |  |</br>"
+          },
+          {
+            "name": "includeAllPolicies",
```

Comment:
>includeAllPolicies [](http://example.com/codeflow?start=21&length=18)

How about using "includeAllPolicies" also within $filter OData query parameter, instead of defining a separate query param?

Reply:
but I want to add Filter for Compliance State Field which is an enum vs include all policies is just an true or false field